### PR TITLE
Added dtHub documentation

### PR DESCRIPTION
**Pull request by amrutaphansalker**
Please review this for accuracy.

### CODE REVIEW

**Comment by amrutaphansalker** [comment]
@davidyan74 @sashadt Please check.

File: docs/dtHub Draft One -- Final.md [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+3. On the menu bar, click **Develop**.
+
+4. Click **import from dtHub**.
![image alt text](image_0.png)

![image alt text](image_1.png)
+
+## Import packages using dtHub
+
+To save the package in the Hadoop cluster where dtGateway is running, use the import option.** **An import operation saves the package on the Hadoop file system.
```
Comment (sashadt) [inline docs/dtHub Draft One -- Final.md]: Some description should be provided.  And images are not consistent in size, and what they are showing.  Second one can be trimmed to just show the console.

File: docs/dtHub Draft One -- Final.md [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+
+1. In the list of packages, locate the package that you want.
+
+2. Click **import**. 
![image alt text](image_2.png)
+
+The import button shows a progress bar indicating the progress of the import operation. After the import operation completes, a notification appears indicating that the import is complete.  
+
+## Download packages using dtHub
+
```
Comment (sashadt) [inline docs/dtHub Draft One -- Final.md]: Image needs to be trimmed to only show the console.

File: docs/dtHub Draft One -- Final.md [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+http://<RTS installation host>:<port defined during installation>/
+
+2. (*Authentication enabled*) Use your credentials to log on. 
+
+3. On the menu bar, click **Develop**.
![image alt text](image_5.png)
+
+4. On the Application Packages page, click **check for updates**.
![image alt text](image_6.png)
+
```
Comment (sashadt) [inline docs/dtHub Draft One -- Final.md]: Needs to be intented to format it as code.

File: docs/dtHub Draft One -- Final.md [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+version: The version of the artifact
+requiresApexVersion: The version of Apache Apex for which artifacts must be retrieved 
+type: The type of the artifact, for example, "apa"
+name: The name of the artifact
+changes: The changes, if any, to the artifact value
+description: A short description of the artifact that you want to retrieve 
+time: The time when the RESTful API command is executed
+tags: Additional tags for enabling search
+requiredArtifacts: The mandatory artifacts
+allVersions: (true | false - default)
```
Comment (sashadt) [inline docs/dtHub Draft One -- Final.md]: Formatting for these parameters doesn't look good.  Needs to be one per line.

File: docs/dtHub Draft One -- Final.md [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+requiresApexVersion: The version of Apache Apex for which artifacts must be retrieved 
+type: The type of the artifact, for example, “apa”
+name: The name of the artifact
+changes: The changes, if any, to the artifact value
+description: A short description of the artifact that you want to retrieve 
+time: The time when the RESTful API command is executed
+tags: Additional tags for enabling search
+requiredArtifacts: The mandatory artifacts</td>
+  </tr>
+  <tr>
```
Comment (sashadt) [inline docs/dtHub Draft One -- Final.md]: Formatting for these parameters doesn't look good. Needs to be one per line.

File: docs/dtHub Draft One -- Final.md [hunk 6]
```diff
@@ -0,0 +1,414 @@
...
+version: The version of the artifact
+requiresApexVersion: The version of Apache Apex for which artifacts must be retrieved 
+type: The type of the artifact, for example, “apa”
+name: The name of the artifact
+changes: The changes, if any, to the artifact value
+description: A short description of the artifact that you want to retrieve 
+time: The time when the RESTful API command is executed
+tags: Additional tags for enabling search
+requiredArtifacts: The mandatory artifacts</td>
+  </tr>
```
Comment (sashadt) [inline docs/dtHub Draft One -- Final.md]: Formatting for these parameters doesn't look good. Needs to be one per line.

File: docs/dtHub Draft One -- Final.md (hunk 1 above)
Comment (amrutaphansalker) [inline docs/dtHub Draft One -- Final.md]: What description are you looking for here?

**Comment by trusli** [comment]
This doc is no longer valid given all the changes in AppHub

**Pull request closed without merging**